.readthedocs.yml

```
@@ -6,13 +6,13 @@ build:
     - libnuma1
 
 python:
-  version: 3.7
+  version: 3.8
   install:
     - method: setuptools
       path: .
 
 conda:
-  environment: conda/environments/builddocs_py37.yml
+  environment: conda/environments/builddocs_py38.yml
```

Maybe we can just rename to drop the Python version for simplicity?

```suggestion
  environment: conda/environments/builddocs.yml
```

---

Good idea, done in https://github.com/rapidsai/ucx-py/pull/829/commits/8cb2dd3c7ec446098eb21a942c85041321cc2a97 .